fetcher/handlers/l0_fetcher.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import subprocess
+import tempfile
+from typing import Any, List, Dict
+
+import boto3
+
+
+BotoClient = Any
+Event = Dict[str, Any]
+
+FTP_PATH = "/pub/OPS/TM/"
+
+
+def get_rclone_config_path(
+    ssm_client: BotoClient,
```

Comment:
You didn't manage to find the client types either, I take it?

Reply:
Nope not for the `boto.client` function. Other than that boto seems typed